NIMATRO/NIMATRO-IOC-01App/Db/controls.db

```diff
@@ -0,0 +1,422 @@
+	field(DRVL, "-174.2")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SPEED:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+alias("$(P)SPEED", "$(P)SPEED:SP:RBV")
+
+# Write to LabVIEW control "Target Pressure" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)PRESSURE:TARGET")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(lvfp,0,0)Target_Pressure")
+    field(PREC, "3")
+    field(SCAN, ".5 second")
+    field(EGU, "mN/m")
+    info(INTEREST, "MEDIUM")
+    info(archive, "VAL")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE:TARGET")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)PRESSURE:TARGET:SP")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
+    field(DTYP, "asynFloat64")
+    field(OUT,  "@asyn(lvfp,0,0)Target_Pressure")
+    field(PREC, "3")
+    field(EGU, "mN/m")
+	field(DRVH, "75.0")
+	field(DRVL, "-75.0")
+    info(INTEREST, "MEDIUM")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE:TARGET:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+alias("$(P)PRESSURE:TARGET", "$(P)PRESSURE:TARGET:SP:RBV")
+
+# Read LabVIEW control/indicator "pressure" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)PRESSURE")
+{
+	field(DESC, "LabVIEW 'pressure'")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(lvfp,0,0)pressure")
+    field(PREC, "3")
+    field(SCAN, ".5 second")
+    field(EGU, "mN/m")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE")
+    field(SDIS, "$(P)DISABLE")
+}
+
+# Write to LabVIEW control "Target  Area" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)AREA:TARGET")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(lvfp,0,0)Target__Area")
+    field(PREC, "3")
+    field(SCAN, ".5 second")
+    field(EGU, "cm^2")
+    info(INTEREST, "MEDIUM")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:AREA:TARGET")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)AREA:TARGET:SP")
+{
+	field(DESC, "LabVIEW 'Target  Area'")
+    field(DTYP, "asynFloat64")
+    field(OUT,  "@asyn(lvfp,0,0)Target__Area")
+    field(PREC, "3")
+    field(EGU, "cm^2")
+	field(DRVH, "247.0")
+    field(DRVL, "36.0")	
+    info(INTEREST, "MEDIUM")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:AREA:TARGET:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+alias("$(P)AREA:TARGET", "$(P)AREA:TARGET:SP:RBV")
+
+# Read LabVIEW control/indicator "area" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+
+# EGU left blank: By default the area is cm^2. However this can be changed on the NIMA Trough.vi but doesn't pass the units
+# to the Nima Trough Interface (which we scan to populate records). 
```

Comment:
This feels like a gotcha. At a minimum it needs documenting on the wiki.

Reply:
Updated the wiki with an explanation. The comment above the record itself should also make this apparent.